### PR TITLE
feat(fgr): update pdf table with user data

### PR DESCRIPTION
**Pull request by pgurusinga**

### CODE REVIEW

File: app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/calculateDuration.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  const diffInMilliseconds = end.getTime() - start.getTime();
+  if (diffInMilliseconds < 0) return "";
+
+  const diffHours = Math.floor(diffInMilliseconds / (1000 * 60 * 60));
+  const diffMinutes = Math.floor(
+    (diffInMilliseconds % (1000 * 60 * 60)) / (1000 * 60),
+  );
+
+  const hourLabel = diffHours <= 1 ? "Stunde" : "Stunden";
```
Comment (pgurusinga) [inline app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/calculateDuration.ts:31]: use 0 Stunden instead

File: app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/__test__/addTable.test.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {
```
Comment (pgurusinga) [inline app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/__test__/addTable.test.ts:1]: change location of test

File: app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/addTable.ts [hunk 3]
```diff
@@ -1,20 +1,48 @@
...
   drawTableColumnsHead(doc, table, startTableY); // Use the table structure element
-  drawTableColumnsValues(doc, table, startTableY); // Continue with table structure element
+  drawTableColumnsValues(doc, table, startTableY, userData); // Continue with table structure element
 
   tableSect.add(table); // Add the table to the section
   documentStruct.add(tableSect); // Add the section to the parent structure
   doc.fill("black"); // Fill black due next pages of the table
 }
+
+export function addTableInfo(
```
Comment (pgurusinga) [inline app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/addTable.ts:28]: move to file

File: app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/drawTableRowHead.ts [hunk 4]
```diff
@@ -1,19 +1,53 @@
+import type { FluggastrechtContext } from "~/domains/fluggastrechte/formular/context";
 import { drawCell } from "./drawCell";
 import {
   COLUMN_HEIGHT,
   COLUMN_WIDTH,
   START_TABLE_X,
 } from "./tableConfigurations";
 
+type FluggastrechteDelayType =
```
Comment (pgurusinga) [inline app/domains/fluggastrechte/services/pdf/sections/reason/factsOfCases/table/drawTableRowHead.ts:9]: use `fluggastBereichSchema`

**Review by aaschlote — APPROVED** [review]
LGTM.

**Pull request merged**